modules/osdk-control-compat.adoc

```diff
@@ -29,19 +29,20 @@ If a cluster administrator has installed your Operator, before they upgrade to t
 
 The following procedure helps prevent administrators from installing versions of your Operator on an incompatible version of {product-title}. These steps also prevent administrators from upgrading to a newer version of {product-title} that is incompatible with the version of your Operator that is currently installed on their cluster.
 
+This procedure is also useful when you know that the current version of your Operator will not work well, for any reason, on a specific {product-title} version. By using it, you define the cluster versions where the Operator should be distributed, and the Operator does not appear in a catalog of a cluster version which is outside of the range.
+
 [IMPORTANT]
 ====
-You should configure the following settings in your Operator project as soon as possible. Users running workloads with deprecated APIs that are planning to eventually upgrade to a future version of {product-title} must be running an Operator that has the proper configuration to avoid a cluster upgrade to an incompatible version and potentially adversely impact their critical workloads.
+If your Operator is using deprecated APIs, you should configure the following settings in your Operator project as soon as possible. Users running workloads with deprecated APIs that are planning to eventually upgrade to a future version of {product-title} must be running an Operator that has the proper configuration to avoid a cluster upgrade to an incompatible version and potentially adversely impact their critical workloads.
 ====
 
 .Prerequisites
 
 - Operator SDK CLI installed on a development workstation
-- Operator project initialized by using the Operator SDK
 
```

Comment:
@adellape we can remove all. that is valid no matter if the project was created or not with SDK. That is a requirement to publish in OCP/OLM

Reply:
As discussed on Slack, changed this to just be:

> An existing Operator project

Reply:
Great feel free to close this one.